tests/Feature/ReservationTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Reservation;
+use App\Models\Meeting;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class ReservationTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_store()
+    {
+        $organizer = factory(User::class)->create();
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $user = factory(User::class)->create();
+
+        $response = $this->actingAs($user, 'api')->postJson(
+            route('api.meeting.reservation.create', ['meeting' => $meeting->getKey()])
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson(['success' => true]);
+    }
+
+    public function test_index()
+    {
+        $organizer = factory(User::class)->create();
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $user1 = factory(User::class)->create();
+        $attendee = new Reservation(['attending' => Reservation::USER_ATTENDING]);
+        $attendee->meeting()->associate($meeting);
+        $user1->reservations()->save($attendee);
+
+        $user2 = factory(User::class)->create();
+        $attendee = new Reservation(['attending' => Reservation::USER_ATTENDING]);
+        $attendee->meeting()->associate($meeting);
+        $user2->reservations()->save($attendee);
+
+        $response = $this->actingAs($organizer, 'api')->getJson(
+            route('api.meeting.reservation.list', ['meeting' => $meeting->getKey()])
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 2
+            ]);
```

Comment:
Can we also assert against the reservation appearing in the json?

Reply:
Yes but I'm not sure you're going to like my solution.